### ISSUE TITLE
Where is the updated sdk source?

### ISSUE THREAD

**Issue by donlzx** [magic-lang/rock#83]
I can compile rock using the latest release (1.0.22) source code or using git master branch, but I can not compile any program with it:  the system.use file is not found.

Also, the sdk folder seems to be not up-to-date.  Many changes noted on the release log are not found in the sdk source folder.  Am I missing something?

**Comment by marcusnaslund** [comment]
AFAIK, the `sdk/` folder is (with minor modifications?) the same as the sdk folder in upstream https://github.com/ooc-lang/rock and used *only* to build rock itself.

Compiled programs are meant to use the (very much updated) sdk found in https://github.com/magic-lang/ooc-kean - there you find a `system/` folder, among other things.

**Comment by marcusnaslund** [comment]
@thomasfanell can perhaps confirm and/or fill in if I missed something.

**Comment by thomasfanell** [comment]
This is correct.

**Comment by donlzx** [comment]
@marcusnaslund  @thomasfanell Thanks for the quick reply.

When I first discover ooc-lang.org, the language design seems very interesting and indeed of some practical use cases.  Sadly its development seems to be stalled.  Then, I found this fork and glad it is still being updated.

However, the documentation is very lacking and the changes/differences from upstream are confusing to a new comer.  Hopefully a simple summary document or one-page website can be setup.  In this way, we may get more people interested.

**Comment by marcusnaslund** [comment]
You are absolutely right, and there have been plans for more extensive documentation, but I'm afraid less and less time from our part is being spent maintaining our ooc fork as well.

If you need any help getting started though, just ask. :) The main differences from "upstream ooc" are the lack of GC and the much more extensive standard library.
